blaze/expr/expressions.py

```diff
@@ -790,6 +804,92 @@ def coerce(expr, to):
     return Coerce(expr, dshape(to) if isinstance(to, _strtypes) else to)
 
 
+class Cast(Expr):
```

Comment:
Yeha, coerce will actually affect the data, cast will just reinterperate the data